docs/secret.rst

```diff
@@ -181,5 +181,5 @@ Reference
 Algorithm details
 -----------------
 
-:Encryption: `Salsa20 stream cipher <https://en.wikipedia.org/wiki/Salsa20>`_
+:Encryption: `XSalsa20 stream cipher <https://libsodium.gitbook.io/doc/advanced/stream_ciphers/xsalsa20>`_
```

Comment:
The change to xsalsa is correct; while I'm unsure the linked page from libsodium is much better than the Wikipedia one,  any reference to the right algorithm is better than the error we had...